app/assets/stylesheets/application.css.scss

```diff
@@ -360,6 +338,41 @@ font-size: 13px
   display: block;
 }
 
+.admin-dash{
+  font-family: 'Oswald', sans-serif;
+  h2{
+    color: #ffab40
+  }
+  a{
+    font-family: 'Oswald', sans-serif;
+    color: #ffab40
+  }
+  a.delete{
```

Comment:
Merge rule `a.delete` with rule on line 346<br>Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space